main.go

```diff
@@ -77,7 +77,11 @@ func main() {
 	}
 
 	if printConfig {
-		fmt.Printf("%s\n", configuration.String())
+		configurationString, err := configuration.String()
+		if err != nil {
+			os.Exit(1)
```

Comment:
Thanks ;) here you go : https://github.com/almighty/almighty-core/pull/712/files#diff-7ddfb3e035b42cd70649cc33393fe32cR82